### ISSUE TITLE
images are not shown

### ISSUE THREAD

**Issue by allamand** [magento/magento2-sample-data#174]
Is that normal that I can't see images ?

![image](https://user-images.githubusercontent.com/8956643/141966945-a6c7c010-8716-46e9-a8b2-8ec409035936.png)

**Comment by atdcloud** [comment]
yeah, i found that it is referencing a cache that is non-existent. haven't yet figured it out how to fix it.
